content/html/concepts/tags/terms/picture/picture.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '<picture>'
+Description: 'Represents multiple possible image sources to be applied to different devices and screen-sizes.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Images'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<picture>` tag represents multiple possible image sources to be applied to different devices and screen-sizes. While it has no semantic meaning, the `<picture>` tag will separate its contents from the rest of the document.
+
+## Syntax
+
+```pseudo
+<picture>
+  Zero or more source tags go here,
+  followed by one image tag here.
+</picture>
+```
+
+`<picture>` is a tag that allows developers the flexibility to use different image specifications.
+
+## Example
+
+The `<picture>` tag is used with zero or more `<source>` elements followed by one [`<img>` element](https://www.codecademy.com/resources/docs/html/images).
```

Comment:
Could we actually move the content from this line up to line 18? It seems like this goes more naturally with the syntax section. 😄

Reply:
@Dusch4593  Took me a few seconds to read over it but yes it does improve the entry and it makes more sense 😄